### PR TITLE
Remove redirects from whitehall

### PR DESCRIPTION
**Pull request by callumknights**
This removes the transformative redirects from Whitehall that were implemented into finder-frontend in [this PR](https://github.com/alphagov/finder-frontend/pull/3050).

Trello: https://trello.com/c/D9Prdm5w/524-remove-redirect-logic-from-whitehall

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -65,14 +65,8 @@ def external_redirect(path_prefix, target)
     resources :fatality_notices, path: "fatalities", only: [:show]
     get "/ministers(.:locale)", as: "ministerial_roles", to: "ministerial_roles#index", constraints: { locale: valid_locales_regex }
     get "/uploads/system/uploads/attachment_data/file/:id/*file.:extension/preview" => "csv_preview#show", as: :csv_preview
-
-    get "/consultations", as: "consultations", to: "consultations#index" # used only for a redirect
-    get "/statistics/announcements", as: "statistics_announcements", to: "statistics_announcements#index" # used only for a redirect
     # End of public facing routes still rendered by Whitehall
 
-    # Routes no longer rendered by Whitehall, but retained to maintain the route helpers
-    get "/statistics(.:locale)", as: "statistics", to: "statistics#index", constraints: { locale: valid_locales_regex }
```
Comment (BeckaL) [inline config/routes.rb:74]: Am I missing something here, or do we actually still have to do the work to do the statistics redirect? This wasn't in your PR (that was /statistics/announcements and /consultations, and the [content item](https://www.gov.uk/api/content/government/statistics) still has a rendering app of whitehall

Comment (callumknights) [reply config/routes.rb:74]: Ah, I have forgotten something. But now what you'd think. Somehow the `statistics` redirect has been hacked straight into router without a content item, so the query params for this haven't been handled for the last two years, and given that as far as me and Bruce could tell this redirect was for non-english finders, which we no longer support, we decided it wasn't very important. What I did forget was to add something into short-url-manager for this (and likely special-routes-publisher) so there is at least a content item. I can do that shortly.

Comment (BeckaL) [reply config/routes.rb:74]: Cool, let me know when done and I can have another look

**Review by brucebolt — APPROVED** [review]
Looks good to me 👍.  Nice to see this code being deleted.

**Pull request merged**